python-avocado.spec

```diff
@@ -459,6 +467,7 @@ Again Shell code (and possibly other similar shells).
 - Add missing configuration files for sub packages
 - Remove man pages source files from package
 - Add bash subpackage
+- Add back examples files into examples package
```

Comment:
Does it mean somewhere in the middle these files were not present in the resulting RPM? That does not sound right, each commit should work independently.

Reply:
Yes, you're right.  I'm squashing some commits that make more sense together on v2.